### PR TITLE
Typos in documentation for coalesce()

### PR DESCRIPTION
**Pull request by dgromer**

### CODE REVIEW

File: R/coalesce.R [hunk 1]
```diff
@@ -1,10 +1,10 @@
 #' Find first non-missing element
 #'
-#' Given a set of vectors, \code{coelesce} finds the first non-missing value
+#' Given a set of vectors, \code{coalesce} finds the first non-missing value
 #' at each position. This is inspired by the SQL \code{COALESCE} function
 #' which does the same thing for \code{NULL}s.
 #'
-#' @param x,... Vectors. All inputs should either be length 1, or the
+#' @param x, ... vectors. All inputs should either be length 1, or the
```
Comment (hadley) [inline R/coalesce.R]: That lack-of-space is important

**Comment by hadley** [comment]
Otherwise, looks great - thanks!

File: R/coalesce.R (hunk 1 above)
Comment (hadley) [inline R/coalesce.R]: And I capitilise the first word - but maybe you were misreading as "... vectors" rather than documenting arguments `x` and `...`?

Comment (dgromer) [inline R/coalesce.R]: I think I was just baffled by the one-word sentence. ;)

**Comment by dgromer** [comment]
Whitespace removed (this is important for roxygen I guess?) and capitalization fixed.

**Pull request merged**